src/gratia_cleanup.py

```diff
@@ -0,0 +1,17 @@
+#!/bin/env python
+
+import os
+import datetime
+
+os.environ['CONDOR_CONFIG'] = '/etc/condor-ce/condor_config'
+
+import htcondor
+
+GRATIA_DIR = htcondor.param['PER_JOB_HISTORY_DIR']
+HISTORY_FILES = [os.path.join(GRATIA_DIR, x) for x in os.listdir(GRATIA_DIR)]
```

Comment:
How much is 'lots and lots'? I just touched 100k files in the history dir, ran the script (modified to remove files older than a second), and the script completed in 2.4s.